pkg/util/testingjobs/raycluster/wrappers.go

```diff
@@ -150,6 +150,11 @@ func (j *ClusterWrapper) WithWorkerPriorityClassName(value string) *ClusterWrapp
 	return j
 }
 
+func (j *ClusterWrapper) WithNumOfHosts(value int32) *ClusterWrapper {
+	j.Spec.WorkerGroupSpecs[0].NumOfHosts = value
```

Comment:
Why "0" index?

Reply:
It's just for testing and following the assumption in other places that the generated object only has 1 worker group

Reply:
I don't like that we hardcode this index. Maybe like this?

```
func (j *ClusterWrapper) WithNumOfHosts(groupName string, value int32) *ClusterWrapper {
	for index, group := range j.Spec.WorkerGroupSpecs {
		if group.GroupName == groupName {
			j.Spec.WorkerGroupSpecs[index].NumOfHosts = value
		}
	}
	return j
}
```